Exploration Queries/InputEntity_Account/UserAccount_Peers.yaml

```diff
@@ -0,0 +1,41 @@
+Id: 18B7E4E3-5B57-4924-B3CD-7E9A5A143521
+DisplayName: Account's peers involved in alerts
+Description: The query located the Account's peers that were involved in a recent alert 
+InputEntityType: Account
+InputFields:
+  - Name
+  - UPNSuffix
+OutputEntityTypes:
+  - Account
+QueryPeriodBefore: 7d
+QueryPeriodAfter: 0d
+DataSources:
+  - UserPeerAnalytics
+  - SecurityAlert
+Tactics:
+  - LateralMovement
+query: |
+
+  let GetUserPeersWithAlerts = (v_Account_Name:string, v_Account_UPNSuffix:string) { 
+    let Account_UPN = strcat(v_Account_Name, '@',v_Account_UPNSuffix);
+    let Peers= UserPeerAnalytics  | where UserPrincipalName == Account_UPN
+        | summarize arg_max(TimeGenerated,*) by PeerUserId | sort by Rank asc
+        | project PeerUserPrincipalName, Rank
+        | extend PeerUserPrincipalName=tolower(PeerUserPrincipalName)
+        | parse PeerUserPrincipalName with Account_Name '@' Account_UPNSuffix;
+    let PeerNames= Peers | summarize make_list(Account_Name);
```

Comment:
Would it make more sense to include the UserId first so we can join more directly later?